src/Commands/DumpCommand.php

```diff
@@ -21,12 +21,18 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $vendorDirectory = $this->getComposer()->getConfig()->get('vendor-dir');
+        $composer = $this->getComposer();
+
+        if ($composer == null) {
```

Comment:
I'm not entirely sure how much this helps, but I guess this can be a strict type check if this change is being made. 👍🏻 🤔
```suggestion
        if ($composer === null) {
```